src/audio/base_fw.c

```diff
@@ -502,6 +542,7 @@ static int basefw_get_large_config(struct comp_dev *dev,
 	case IPC4_SCHEDULERS_INFO_GET:
 	case IPC4_GATEWAYS_INFO_GET:
 	case IPC4_LIBRARIES_INFO_GET:
+		return basefw_libraries_info_get(data_offset, data);
```

Comment:
you need a `break` after line 543 here too.

Reply:
Good point, added break